3rd-party-licenses.md

```diff
@@ -0,0 +1,147 @@
+License: MIT  
+https://github.com/eslint/eslint/blob/master/LICENSE
+
+### jQuery
+License: MIT or GPL Version 2  
+http://jquery.org/license
+
+### selectize.js
+License: Apache License 2.0  
+https://github.com/selectize/selectize.js/blob/master/LICENSE
+
+### slick.js
+License: MIT  
+https://github.com/kenwheeler/slick/blob/master/LICENSE
+
+### nodejs-npm
+License: Artistic License 2.0  
+https://www.npmjs.com/policies/npm-license
+
+### gnu-libiconv
+License: GPL v3  
+https://directory.fsf.org/wiki/Libiconv
+
+### eslint-config-standard
+License: MIT  
+https://github.com/standard/eslint-config-standard/blob/master/LICENSE
+
+### Autoprefixer
+License: MIT  
+https://github.com/postcss/autoprefixer/blob/master/LICENSE
+
+### ES6-Promise (subset of rsvp.js)
+License: MIT  
+https://github.com/stefanpenner/es6-promise/blob/master/LICENSE
+
+### time-grunt
+License: MIT  
+https://github.com/sindresorhus/time-grunt/blob/master/license
+
+### jit-grunt 0.10.0
+License: MIT  
+https://github.com/shootaroo/jit-grunt
+
+### SVG to webfont converter for Grunt
+License: MIT  
+https://github.com/sapegin/grunt-webfont/blob/master/License.md
+
+### grunt-spritesmith
+License: MIT  
+https://github.com/twolfson/grunt-spritesmith/blob/master/LICENSE-MIT
+
+### grunt-postcss
+License: MIT  
+https://github.com/nDmitry/grunt-postcss/blob/master/LICENSE
+
+### grunt-legacssy
+License: MIT  
+https://github.com/robinpokorny/grunt-legacssy/blob/master/LICENSE
+
+### eslint-plugin-standard
+License: MIT  
+https://github.com/standard/eslint-plugin-standard/blob/master/LICENSE
+
+### eslint-plugin-promise
+License: ISC  
+https://github.com/xjamundx/eslint-plugin-promise#license
+
+### eslint-plugin-node
+License: MIT  
+https://github.com/mysticatea/eslint-plugin-node/blob/master/LICENSE
+
+### eslint-plugin-import
+License: MIT  
+https://github.com/benmosher/eslint-plugin-import/blob/master/LICENSE
+
+## Images and components used to run Shopsys Framework on Docker
+### Postgres
+License: PostgreSQL License  
+https://www.postgresql.org/about/licence/
+
+### Nginx
+License: BSD-2-Clause  
+http://nginx.org/LICENSE
+
+### Redis
+License: BSD-3-Clause  
+https://redis.io/topics/license
+
+### Erikdubbelboer/phpredisadmin
+License: Creative Commons Attribution 3.0  
+https://github.com/erikdubbelboer/phpRedisAdmin
+
+### Selenium/standalone-chrome
+License: Apache License 2.0  
+https://github.com/SeleniumHQ/docker-selenium/blob/master/LICENSE.md
+
+### Adminer
+License: Apache License 2.0 or GPL 2  
+https://github.com/vrana/adminer/blob/master/readme.txt
+
+### Docker.elastic.co/elasticsearch/elasticsearch-oss
+License: Apache License 2.0  
+https://github.com/elastic/elasticsearch/blob/66b5ed86f7adede8102cd4d979b9f4924e5bd837/LICENSE.txt
+
+### Php
+License: The PHP License  
+https://hub.docker.com/_/php/
+
+## Libraries dynamically referenced via Composer
+Run `composer license` in your project to get the latest licensing info about all dependencies. 
```

Comment:
There should probably be a note that you should run this inside the `php-fpm` containers as well as both microservices, they might use additional packages and libraries.